src/pages/Editor.vue

```diff
@@ -888,6 +899,32 @@ export default {
     ]),
     ...mapActions("generic", ["showSharePlioDialog"]),
     ...Utilities,
+    throwConfetti() {
+      var end = Date.now() + 10 * 1000;
```

Comment:
one comment on what is end here? - guessing that this is the time when the animation should end.

Reply:
animationEndTime